src/Home.tsx

```diff
@@ -1,85 +1,149 @@
-      <HorizontalScroll>
-        {SLIDER_CARDS.map((card, index) => (
-          <>
-            <SliderCard
-              decorationLine={card.decorationLine}
-              title={card.title}
-            />
-            {index !== SLIDER_CARDS.length && <HorizontalSeparator />}
-          </>
-        ))}
-      </HorizontalScroll>
-      <VerticalSeparator />
-      <HorizontalScroll>
-        {SLIDER_CARDS.map((card, index) => (
-          <>
-            <SliderCard
-              decorationLine={card.decorationLine}
-              title={card.title}
-            />
-            {index !== SLIDER_CARDS.length && <HorizontalSeparator />}
-          </>
-        ))}
-      </HorizontalScroll>
-      <div style={{ height: '100px' }} />
-      <ServiceCard
-        title={'Storming'}
-        description={
-          'We help innovators quickly understand how we will build their digital products and provide a competitive business advantage.'
-        }
-        strongLastPhrase={'The idea that works.'}
-        color={Gradients.blue}
-      />
-      <ServiceAspect
-        color={Colors.purple}
-        icon={BenchmarkIcon}
-        title={'Benchmark'}
-        description={
-          'Start learning from the experiences of others. That could lead us to find out about existing solutions to similar problems, and accelerate product ideation pace.'
-        }
-      />
+const StyledHome = styled.div`
+  background-color: ${Colors.black};
+`;
+
+const StyledHeaderTitle = styled.div`
+  padding-left: 64px;
+  padding-top: 40px;
+  font-size: 88px;
+  font-weight: bold;
+  color: white;
 
-      <div style={{ height: '40px', width: '100%' }} />
+  @media (max-width: 767px) {
+    font-size: 44px;
+  }
+`;
 
-      <Row style={{ margin: '0' }}>
-        <Col xs={6} style={{ padding: '0' }}>
-          <ServiceNavButton title={'Augmentation'} color={Gradients.green} />
+const StyledCaption = styled.div`
+  text-align: center;
+  font-size: 16px;
+  font-weight: bold;
+  color: white;
+  opacity: 0.24;
+
+  @media (max-width: 767px) {
+    font-size: 44px;
+  }
+`;
+
+const StyledImage = styled.div`
+  text-align: right;
+`;
+
+export const Home = () => {
+  return (
+    <StyledHome>
+      <Row middle={'xs'}>
+        <Col xs={4} sm={6}>
+          <StyledHeaderTitle>
+            Lean digital innovation that works.
+          </StyledHeaderTitle>
         </Col>
-        <Col xs={6} style={{ padding: '0' }}>
-          <ServiceNavButton title={'Delivery'} color={Gradients.yellow} next />
+        <Col xs={8} sm={6}>
+          <StyledImage>
+            <img src={TypeLogo} alt={''} width={'100%'} />
+          </StyledImage>
         </Col>
       </Row>
+      <HorizontalScroll>
+        <div
+          style={{
+            padding: '40px',
+            display: 'flex',
+            justifyContent: 'space-between',
+            width: '100%',
+            minWidth: '1200px',
+          }}
+        >
+          <div className={'column'}>
```

Comment:
Seria bom mudar isso pra Col do grid... se for deixar pra dps eh bom abrir uma issue pra n esquecermos